gloss/Graphics/Gloss/Geometry/Triangulation.hs

```diff
@@ -0,0 +1,388 @@
+module Graphics.Gloss.Geometry.Triangulation
+        (triangulate)
+where
+        --TODO put exported functions here. triangulate only to be exact. pozdrav!
+import Graphics.Gloss.Rendering
+import Graphics.Gloss.Geometry.Line
+import Graphics.Gloss.Geometry.Angle
+import Graphics.Gloss.Data.Vector
+import qualified Data.Set as Set
+import qualified Data.Map.Strict as Map
+import Data.List (minimumBy, maximumBy, sortBy)
+import Control.Applicative
+import Data.Maybe
+
+-- import Debug.Trace
+
+
+-- | Type for representing planar graphs.
+type Vertices = Map.Map Vertex (Set.Set Vertex)
+
+-- | Whrepper for @Point@. Has different @Eq@ implmentation to absorb numerical error in calculations.
+data Vertex
+        = Vertex Point
+        deriving (Show)
+
+-- | Utility function to convert @Vertex@ to @Point@
```

Comment:
You don't need to say "Utility function" in these comments, just write "Convert a Vertex to a Point". We can already see that it is a utility function.